### PR TITLE
User stopped typing behavior

### PR DESCRIPTION
**Pull request by CliffAgius**
Updated the description in the Index to match that in the Doc.

### CODE REVIEW

**Comment by PRMerger15** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Index File | This PR contains a change to the index file.

File: docs/maui/behaviors/index.md [hunk 1]
```diff
@@ -20,4 +20,4 @@ The .NET MAUI Community Toolkit provides a collection of pre-built, reusable beh
 | Converter | Description |
 | --------- | ----------- |
 | [`MaxLengthReachedBehavior`](maximum-length-reached-behavior.md) | The `MaxLengthReachedBehavior` is a behavior that allows the user to trigger an action when a user has reached the maximum length allowed on an `InputView`. |
-| [`UserStoppedTypingBehavior`](user-stopped-typing-behavior.md) | The `UserStoppedTypingBehavior` is a behavior that allows the user to trigger an action when a user has stopped data input an `Entry`. |
+| [`UserStoppedTypingBehavior`](user-stopped-typing-behavior.md) | TThe `UserStoppedTypingBehavior` is a `Behavior` that will trigger an action when a user has stopped data input on controls for example `Entry`, `SearchBar` and `Editor`. Examples of its usage include triggering a search when a user has stopped entering their search query. |
```
Comment (jfversluis) [inline docs/maui/behaviors/index.md:23]:
```suggestion
| [`UserStoppedTypingBehavior`](user-stopped-typing-behavior.md) | The `UserStoppedTypingBehavior` is a `Behavior` that will trigger an action when a user has stopped data input on controls for example `Entry`, `SearchBar` and `Editor`. Examples of its usage include triggering a search when a user has stopped entering their search query. |
```

Comment (jfversluis) [reply docs/maui/behaviors/index.md:23]: Typo!

File: docs/maui/behaviors/user-stopped-typing-behavior.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: UserStoppedTypingBehavior - .NET MAUI Community Toolkit
 author: cliffagius
-description: "The UserStoppedTypingBehavior is a Behavior that allows the user to trigger an action when a user has stopped data input an Entry. Examples of its usage include triggering a search when a user has stopped entering their search query."
+description: "The `UserStoppedTypingBehavior` is a `Behavior` that will trigger an action when a user has stopped data input on controls for example `Entry`, `SearchBar` and `Editor`. Examples of its usage include triggering a search when a user has stopped entering their search query."
```
Comment (jfversluis) [inline docs/maui/behaviors/user-stopped-typing-behavior.md:4]:
```suggestion
description: "The UserStoppedTypingBehavior is a Behavior that will trigger an action when a user has stopped data input on controls for example Entry, SearchBar and Editor. Examples of its usage include triggering a search when a user has stopped entering their search query."
```

The markdown doesn't have any added value here.

**Review by jfversluis — APPROVED** [review]

**Comment by PRMerger12** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Index File | This PR contains a change to the index file.

**Comment by PRMerger12** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Pull request merged**